CMakeLists.txt

```diff
@@ -124,6 +125,13 @@ ecbuild_add_option( FEATURE BUDDY_MALLOC
     DEFAULT ON
 )
 
+## Field Gang
+ecbuild_add_option( FEATURE FIELD_GANG
+    DESCRIPTION "Enable packed storage of fields"
+    DEFAULT ON
+    CONDITION (NOT CMAKE_Fortran_COMPILER_ID MATCHES "Cray")
```

Comment:
Same here, should we mention this in the "supported features" section of the README?